include/Typedef.h

```diff
@@ -379,10 +379,11 @@ const GREP_PresScheme_t
 
 typedef int GREP_Center_t;
 const GREP_Center_t
-   GREP_CENTER_BOX  = 1,
-   GREP_CENTER_DENS = 2,
-   GREP_CENTER_POT  = 3,
-   GREP_CENTER_COM  = 4;
+   GREP_CENTER_BOX     = 1,
+   GREP_CENTER_DENS    = 2,
+   GREP_CENTER_POT     = 3,
+   GREP_CENTER_COM     = 4,
+   GREP_CENTER_RESTART = 5;
```

Comment:
I prefer to call it `GREP_CENTER_NONE = 0` to indicate that `GREP_Center` is not assigned (e.g., is set when reading the restart file).